Data/pickle_tags.py

```diff
@@ -0,0 +1,36 @@
+from lxml import etree as ET
+import tqdm
+import sys
+
+
+def convert(xml_file, output_file):
+    """ Extract the annotations from pubtator xml formatted file
+
+    Keywords arguments:
+    xml_file -- The path to the xml data file
+    output_file -- the path to output the formatted data
+    """
+    with open(output_file, "w") as g:
+        g.write("Document\tType\tID\tOffset\tEnd")
```

Comment:
Try to never hardcode file parsing. Use buitlin `csv` module instead. This way you specify the delimiter (`\t`) once, and pass the `csv.writer` lists for each row. Less error prone when you let the machine do the repetitive work.